.github/workflows/deploy-js-kit-client.yml

```
@@ -31,37 +36,74 @@ env:
     name: Programs
     uses: ./.github/workflows/build-programs.yml
     secrets: inherit
-
-  test_js_client:
-    name: JS client
-    needs: build_programs
-    uses: ./.github/workflows/test-js-client.yml
+    needs: check_tag
+    if: needs.check_tag.outputs.type == 'ref'
+    with:
+      git_ref: ${{ inputs.git_ref }}
```

actionlint (1.7.11)

[error] input "git_ref" is not defined in "./.github/workflows/build-programs.yml" reusable workflow. defined inputs are "rust", "solana"

(workflow-call)